frontend/public/components/build-config.tsx

```diff
@@ -62,12 +64,12 @@ const menuActions = [
 
 export const BuildConfigsDetails: React.SFC<BuildConfigsDetailsProps> = ({ obj: buildConfig }) => {
   const hasPipeline = buildConfig.spec.strategy.type === BuildStrategyType.JenkinsPipeline;
-
+  const { t } = useTranslation();
   return (
     <>
       <div className="co-m-pane__body">
         {hasPipeline && <PipelineBuildStrategyAlert obj={buildConfig} />}
-        <SectionHeading text="Build Config Details" />
+        <SectionHeading text={t('build-config~Build Config Details')} />
```

Comment:
```suggestion
        <SectionHeading text={t('build-config~Build Config details')} />
```

Reply:
@spadgett Should the section heading be the same with breadcrumb?
<img width="978" alt="Screen Shot 2020-10-15 at 1 45 01 PM" src="https://user-images.githubusercontent.com/20943765/96082403-bfa17680-0eed-11eb-973c-15ad8040fc5b.png">

Reply:
The breadcrumbs should be "Build Config details," too. That change will be done in #6888.

Reply:
Got it. Thanks.